src/types.ts

```diff
@@ -202,3 +210,5 @@ export type IExportableAnswerMetadata =
   IExportableMultipleChoiceAnswerMetadata;
 
 export interface LTIRuntimeAnswerMetadata extends ILTIPartial, IExportalbleAnswerMetadataBase { }
+
+export interface AnonymousRuntimeAnswerMetadata extends IAnonymousMetadataPartial, IExportalbleAnswerMetadataBase { }
```

Comment:
```suggestion
export interface AnonymousRuntimeAnswerMetadata extends IAnonymousMetadataPartial, IExportableAnswerMetadataBase { }
```
although if this is imported from elsewhere it may not be fixable here.

Reply:
:+1: